packages/manager/modules/telecom-spare/src/modems/replace/modems-replace.html

```diff
@@ -0,0 +1,24 @@
+<oui-modal data-heading="{{ ::$ctrl.title | translate}}"
```

Comment:
Imo you should either have a binding that you directly display (meaning you provide the already translated content) or you have a static key that you translate